### PR TITLE
Issue #191 - Edit Explore Themes Link

### PR DESCRIPTION
**Pull request by FrancescoSTL**
Issue ref: #191 

The issue is referring to more than just editing the link, but we solved all of that in #205 

@chrismore r?

### CODE REVIEW

File: data/tmpl/utility/content3.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
           <button class="button" id="template2">Try Shiretoko Fox</button>
         </div>
         <button class="button reset" id="default">Reset to Default</button>
-        <a href="https://addons.mozilla.org/firefox/themes/?utm_source=fxonboarding&amp;utm_medium=sidebar&amp;utm_campaign=onboardingv1&amp;utm_content=customization" target="_blank">Explore other themes</a>
+        <a href="https://addons.mozilla.org/firefox/themes/?utm_source=fxonboarding&utm_medium=referral&utm_campaign=onboardingv1" target="_blank">Explore other themes</a>
```
Comment (chrismore) [inline data/tmpl/utility/content3.html]: @FrancescoSTL wrong medium

URL should be:

https://addons.mozilla.org/firefox/themes/?utm_source=fxonboarding&utm_medium=firefox-browser&utm_campaign=onboardingv1

I just mentioned this in a comment in #191

Comment (schalkneethling) [inline data/tmpl/utility/content3.html]: Also escape `&`

**Comment by FrancescoSTL** [comment]
@schalkneethling r?

**Comment by schalkneethling** [comment]
r+ 🔣

**Pull request merged**